syntax = "proto3";

message Inventory{
    int32 id  = 1;
    int64 created_at = 2; 
    int64 updated_at = 3;
    string product_name = 4;
    int32 stock_level = 5;
    }